CHANGES

```diff
@@ -0,0 +1,15 @@
+Hello everyone,
```

Comment:
Where'd this come from?

Reply:
Whoops - this and the example file were due to a dirty workspace.